azurerm/resource_arm_loadbalancer_rule.go

```diff
@@ -98,6 +98,12 @@ func resourceArmLoadBalancerRule() *schema.Resource {
 				Default:  false,
 			},
 
+			"disable_outbound_snat": {
```

Comment:
could we also [add this to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/loadbalancer_rule.html.markdown#L67)?